src/test-reader/test-object/test.ts

```diff
@@ -8,6 +8,7 @@ type TestOpts = TestObjectData &
 
 export class Test extends ConfigurableTestObject {
     public fn: TestFunction<TestFunctionCtx>;
+    public err?: Error;
```

Comment:
Where it is initialized? Does this field exist only in the test instance, or can be in hooks too?

Reply:
Here: https://github.com/gemini-testing/testplane/blob/f9239c9c1985f38b01d7d5fd0dd387fab24fc3c8/src/runner/test-runner/regular-test-runner.js#L40

It can not be in hooks: https://github.com/gemini-testing/testplane/blob/master/src/worker/runner/test-runner/index.js#L153